docs/start-deployment.md

```diff
@@ -155,16 +170,25 @@ GOOGLE_MAPS_API_KEY=my_google_maps_api_key
 
 The `.env` file contains some additional variables to set up social logins, or integrate some extra services. Over time, more values will be added.
 
-## 6. Initialize the database
+## 6. Pull images
+
+This step may take a while since it will download all images needed to run the project.
+
+[Official docs](https://docs.docker.com/compose/reference/pull/)
+```bash
+$ docker-compose pull
+```
+
+## 7. Initialize the database
 
-Almost ready for action. When all configuration is provided, we're ready to setup the database. This step may take a while since it will download all images needed to run the project.
+Almost ready for action. When all configuration is provided, we're ready to setup the database.
 
 ```bash
 $ docker-compose run --rm web rake db:create db:migrate db:seed
```

Comment:
would be nice if we could run seeds with a verbose flag, right now it just seems to hang after the last migration